plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -338,17 +339,23 @@ public PluginCompatReport testPlugins()
 	}
 
     private void generateHtmlReportFile() throws IOException {
+        if (!config.reportFile.exists() || !config.reportFile.isFile()) {
+            throw new FileNotFoundException("Cannot find the XML report file: " + config.reportFile);
+        }
+
         Source xmlSource = new StreamSource(config.reportFile);
-        Source xsltSource = new StreamSource(getXslTransformerResource().getInputStream());
-        Result result = new StreamResult(PluginCompatReport.getHtmlFilepath(config.reportFile));
+        try(InputStream xsltStream = getXslTransformerResource().getInputStream()) {
```

Comment:
How is this related?

Reply:
not related, just fixed the leak while I was around IIRC